examples/basics/demo_dr_methods_affinity_matcher.py

```diff
@@ -0,0 +1,111 @@
+r"""
+Comparison of different DR methods and the use of affinity matcher
+========================================
+
+We illustrate the basic usage of TorchDR with different DR methods on the swiss roll dataset.
+
+"""
+# %%
+import matplotlib.pyplot as plt
+from sklearn.datasets import make_swiss_roll
+from torchdr import AffinityMatcher
+from torchdr.affinity import EntropicAffinity, NormalizedGaussianAffinity, NormalizedStudentAffinity
+from torchdr import SNE, UMAP, TSNE
+import torch
```

Comment:
import order: put all import from other packages in one block, then one blank line, then all imports from torchdr